packages/eventemittable/__tests__/eventemittable.js

```diff
@@ -0,0 +1,197 @@
+var EventEmitter = require('../node_modules/events').EventEmitter;
```

Comment:
That's probably unnecessary, right? :) `require('events').EventEmitter` should work too.